### PR TITLE
Use more inclusive language

### PR DESCRIPTION
**Pull request by jamesobutler**
This PR follows recommendations from https://tools.ietf.org/id/draft-knodel-terminology-09.html which have been adopted by many others in the community for work towards the greater good. Although it is often nice to maintain using current terminology, as it relates to improving inclusivity, this should be higher priority.

This PR includes acting upon what was originally described by @jcfr at https://discourse.slicer.org/t/2021-04-06-hangout/16953/3?u=jamesobutler which detailed:

> Removal of divisive terminology regarding branch name associated with repository hosted in the Slicer and commontk GitHub organization. We will rename `master` to `main`

Renaming of the `master` branch to `main` had been indicated in April 2021, but has not yet been acted upon, until now...

GitHub has improved the ability to rename a branch (https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/renaming-a-branch) which makes it easy to switch to a new name with minimal consequences. This has to be completed by the admin of the Slicer repository (@jcfr).
Some action items:
- [x] Rename default branch for https://github.com/Slicer/Slicer/tree/master
- [x] Rename default branch for https://github.com/Slicer/ExtensionsIndex/tree/master, re https://github.com/Slicer/ExtensionsIndex/pull/1812
- [x] Rename default branch for https://github.com/Slicer/apidocs.slicer.org/tree/master, re https://github.com/Slicer/apidocs.slicer.org/pull/3

Renaming of the default branch looks something like this:
image

### CODE REVIEW

**Comment by jcfr** [comment]
Thanks for working on this :pray: , this is outstanding :100: 

Considering the following:

> When you rename a branch on GitHub.com, any URLs that contain the old branch name are automatically redirected to the equivalent URL for the renamed branch. 

I have no issue moving forward with the rename. I referenced this PR in tomorrow's hangout agenda.

**Comment by jamesobutler** [comment]
I've rebased this PR to fix merge conflicts.

**Comment by jcfr** [comment]
* [x] Branch associated with https://github.com/Slicer/DashboardScripts has been renamed. Relevant crontab files and scheduled tasks have also been updated.

**Comment by jamesobutler** [comment]
I've rebased and updated some additional references for new code.

**Comment by jamesobutler** [comment]
@jcfr Can we proceed with this PR towards using more inclusive language?

**Review by pieper — DISMISSED** [review]
Yes, I think we need to do this.

Anyone want to comment on what timing would be convenient?  Before or after project week for example?  I lean to before so people can get help during project week if needed.

**Comment by lassoan** [comment]
This may be a disruptive change, so it would be nice to announce it on the forum and give a few days for people to comment on it. But if we wait a few days then we'll get too close to the project week. So, I think it would be safer to do it a few days after the project week. That would also give us a chance to close a number of pull requests.

**Comment by jamesobutler** [comment]
Today the United States is observing Federal Holiday [Juneteenth](https://en.wikipedia.org/wiki/Juneteenth), where this would be a very relevant time to integrate this change.

**Comment by lassoan** [comment]
I don't have strong opinion about delaying the change or not. For me it should not be a big difference when this happens. However, if we wan to give notice to Slicer developers then we need to post on the Slicer forum and give at least 1-2 workdays time to respond - ask questions, request changes to the plan, etc. The discourse post would also be a much more visible gesture than merging a pull request.

@jamesobutler would you write a post on discourse that explains what/why/when we plan to do?

**Comment by jamesobutler** [comment]
https://discourse.slicer.org/t/slicer-to-use-more-inclusive-language-in-code/23972 has been posted on the forum with the announcement.

**Comment by jamesobutler** [comment]
@jcfr I will need your help on Thursday to complete this process with you using your GitHub permissions to actually change the default branch name from `master` to `main`.

**Comment by jcfr** [comment]
I will start with the renaming and merging in the next hour or so.

File: Applications/SlicerApp/Testing/Python/SlicerRestoreSceneViewCrashIssue3445.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 class SlicerRestoreSceneViewCrashIssue3445(ScriptedLoadableModule):
     """Uses ScriptedLoadableModule base class, available at:
-    https://github.com/Slicer/Slicer/blob/master/Base/Python/slicer/ScriptedLoadableModule.py
+    https://github.com/Slicer/Slicer/blob/mainase/Python/slicer/ScriptedLoadableModule.py
```
Comment (Sunderlandkyl) [inline Applications/SlicerApp/Testing/Python/SlicerRestoreSceneViewCrashIssue3445.py:12]: Not a big issue, but there are 13 places in comments where "master/Base" was renamed to "mainase".

Comment (jamesobutler) [reply Applications/SlicerApp/Testing/Python/SlicerRestoreSceneViewCrashIssue3445.py:12]: Looks like a mistake I made during rebase. I can fix

Comment (jamesobutler) [reply Applications/SlicerApp/Testing/Python/SlicerRestoreSceneViewCrashIssue3445.py]: Update pushed to fix this.

**Comment by jcfr** [comment]
@Sunderlandkyl thanks for the review 🙏

**Comment by jcfr** [comment]
Additional tasks performed:
* [x] Renamed branch in https://github.com/Slicer/SlicerBuildEnvironment
* [x] Renamed branch in https://github.com/Slicer/DashboardScripts/pull/46
* [x] Connected to `computron`, `metroplex`, `overload` and `factory-south-macos` to update references (`master` -> `main`) for the existing chekckout of `DasboardScripts`, `Slicer` and `SlicerBuildEnvironment`

**Comment by jcfr** [comment]
Additional tasks performed:
* [x] Renamed branch in https://github.com/KitwareMedical/SlicerCustomAppTemplate/pull/36

**Comment by lassoan** [comment]
@jcfr is the `nightly-master` branch still used for something? Can we just remove it? If not then I guess it should be renamed, too.

**Comment by jcfr** [comment]
Thanks for the reminder, I will  have a look later and either remote or make sure it stays up to date.

**Comment by jcfr** [comment]
Additional tasks performed:
* [x] Renamed branch in https://github.com/Slicer/SlicerDockerUpdate/pull/2
* [x] Renamed branch in https://github.com/Slicer/SlicerDocker/pull/50
* [x] Connected to `metroplex` to update references  (`master` -> `main`) for existing checkout in `/home/kitware/Packaging/SlicerDockerUpdate`

**Comment by jcfr** [comment]
> is the nightly-master branch still used for something? Can we just remove it? If not then I guess it should be renamed, too.

It is a reference updated daily pointing to the commit that was used to build the latest set of installers.

The branch has been renamed, and the associated script & crontab have been fixed and updated in https://github.com/Slicer/DashboardScripts/pull/47

**Pull request merged**